registration/views.py

```diff
@@ -187,6 +188,10 @@ def strip_unicode_from_list(unicode_list):
     return stripped_list
 
 
+def csrf_failure(request, reason=""):
```

Comment:
@nasief Why do we need `reason` parameter? can't we just get rid of it? I don't see it being used anywhere around.